runtime/compiler/control/JITClientCompilationThread.cpp

```diff
@@ -256,9 +256,10 @@ handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe, JITServer::Mes
       case MessageType::VM_getSystemClassFromClassName:
          {
          auto recv = client->getRecvData<std::string, bool>();
+         TR_J9VMBase *fej9 = TR_J9VMBase::get(vmThread->javaVM->jitConfig, vmThread);
```

Comment:
Yeah, we need non-AOT front-end here, since validation is done on the server. I mention it in the commit message, will add comment here as well.

Reply:
Ironically, I just reviewed an item from @dchopra001 where I suggested using a TR_J9 frontend in this very place. In a sense I answered my own question in that PR.

Reply:
Can we remove this line of code from here and merge it as part of #10688 instead? The change here is needed to functional correctness on Z. Since we are trying to get this change into the 0.23 branch, it'll be easier to push it through as part of #10688. Unless we are sure that the change in this PR should also be double delivered?

Reply:
I will remove it.

Reply:
Thanks!